CMakeLists.txt

```
@@ -131,6 +131,25 @@ option(
   "build the finite difference based space charge solver, requires PETSc & PkgConfig"
   OFF)
 
+# OpenPMD
+find_package(openPMD CONFIG)
+
+# if not found, fetch it
+if(NOT openPMD_FOUND)
+  include(FetchContent)
+  set(CMAKE_POLICY_DEFAULT_CMP0077 NEW)
+  set(openPMD_BUILD_CLI_TOOLS OFF)
+  set(openPMD_BUILD_EXAMPLES OFF)
+  set(openPMD_BUILD_TESTING OFF)
+  set(openPMD_INSTALL OFF)
```

Maybe more usable for the superbuild is:
```suggestion
  set(openPMD_BUILD_SHARED_LIBS OFF)
  set(openPMD_INSTALL OFF)
```
that way you don't need to install and find a shared lib for openPMD.

---

For the issue with
```suggestion
  set(openPMD_INSTALL ON)
```
I think the issue we see here might be due to the CMake version in CI, I cannot immediately see the same issue on my Ubuntu with CMake `3.23.2`. What is the CMake version in CI here?
```
CMake Error at /home/runner/work/synergia2/synergia2/build/_deps/openpmd-src/CMakeLists.txt:1080 (install):
  install EXPORT given no DESTINATION!


CMake Error at /home/runner/work/synergia2/synergia2/build/_deps/openpmd-src/CMakeLists.txt:1085 (install):
  install FILES given no DESTINATION!
```
https://github.com/openPMD/openPMD-api/blob/78baa4cc16b04b446787e69059f2b7b27aba136f/CMakeLists.txt#L1080-L1085
I will check more...

---

CI: CMake 3.22.1 per https://packages.ubuntu.com/jammy/cmake

---

Fix coming via https://github.com/fnalacceleratormodeling/synergia2/pull/100